docs/computing/containers/creating.md

```diff
@@ -42,3 +42,37 @@ Also see our [documentation on how to run containers](run-existing.md).
 You can also build your own container from scratch. This is an option for more experienced users, and your main source of information is the [official Apptainer documentation on building containers](https://apptainer.org/docs/user/main/build_a_container.html).
 
 You can find some help also by looking at our [tutorial on building Apptainer containers from scratch](../../support/tutorials/singularity-scratch.md).
+
+## Building a container without sudo access
+
+Root access into the cluster is not permitted. Namespaces have also been disabled due to security issues involved. However with a few restrictions Apptainer can still be used by an unprivileged user to build a container by using [fakeroot](https://apptainer.org/docs/user/main/fakeroot.html).
+
+In the cluster Apptainer enables `--fakeroot` flag by default when building containers, this makes the user appear as root:root while building the container thus enabling the user to build images that require root permissions e.g. to install packages via apt.
+However this only makes the user appear as the root user, in the host system user has no additional permissions. By itself fakeroot is not always sufficient and building some containers may fail due to various different reasons. For more details see [Apptainer documentation](https://apptainer.org/docs/user/main/fakeroot.html).
+
+
+The following simple example definition file (saved as `ubuntu.def`) creates an image based on ubuntu with one package installed.
+```
+Bootstrap: docker
+From: ubuntu:22.04
+%post
+	apt-get update
+	apt-get install -y cowsay
+```
+Image can then be built `apptainer build ubuntu.sif ubuntu.def` and ran `apptainer shell ubuntu.sif`. Now the installed package can be accessed in the shell opened by typing `echo hello | /usr/games/cowsay`. Note that sudo is not required to run these commands.
+
+
+Common docker base images and whether installing simple packages with package manager works on them in the cluster:
+
+|Image|Tag|Works|
+|-----|---|-----|
+|alpine|3.6-3.19|no|
+|almalinux|8-9|yes|
+|debian|buster-trixie|yes|
+|centos|7|yes|
+|opensuse/leap|15.0,15.6|no|
+|opensuse/leap|15.1-15.5|yes|
+|redhat/ubi|8-9|yes|
+|ubuntu|16.04-22.04|yes|
```

Comment:
Perhaps it would be good to add which package manager (command) was tested here?

Reply:
You may choose how you want to do it, but I'd suggest being explicit. If it feels redundant to include the commands, then maybe just change "package manager" to "the default package manager"